apps/oxlint/src/output_formatter/junit.rs

```diff
@@ -72,25 +75,26 @@ fn format_junit(diagnostics: &[Error]) -> String {
                 description,
                 severity
             );
-            let test_case =
-                format!("\n        <testcase name=\"{rule}\">\n{status}\n        </testcase>");
-            test_cases = format!("{test_cases}{test_case}");
+            let test_case = format!(
+                "\n        <testcase name=\"{rule}\">\n{status}\n        </testcase>"
+            );
+            test_cases.push_str(&test_case);
```

Comment:
Slight perf to reduce allocations compared to rebuilding the new `String` in `format!` each time. append in place only now